functions.sh

```diff
@@ -32,14 +32,14 @@ function get_arch() {
     s390x)
       arch="s390x"
       ;;
-    aarch64)
+    aarch64 | arm64)
       arch="arm64"
       ;;
     armv7l)
       arch="arm32v7"
       ;;
     *)
-      echo "$0 does not support architecture ${arch} ... aborting"
+      echo "$0 does not support architecture ${arch:-unknown} ... aborting"
```

Comment:
Don't we need this variable printed instead of "unknown" to help digging?

Reply:
point is to avoid the `unbound variable` error that's currently printed

Reply:
oh yes, I misunderstood.